src/otx/engine/engine.py

```diff
@@ -528,14 +528,16 @@ def explain(
         checkpoint: PathLike | None = None,
         datamodule: EVAL_DATALOADERS | OTXDataModule | None = None,
         explain_config: ExplainConfig | None = None,
+        dump: bool | None = False,
```

Comment:
I'd suggest calling it `output`, so if it's not None, it will be both a flag to dump saliency maps and path where to dump them.

Reply:
I would propose to stick to `self.work_dir / "saliency_maps"` as the default place for storing saliency maps. What about `dump` as as saving flag, and `output` as an optional path?

Reply:
I think we can leave it as it is